### PR TITLE
Qingwei/statement ws

### PR DESCRIPTION
**Pull request by qingweibinary**
1. moved all static related scss to /src/scss/static/
2. added new page for statement ws

Note: 
 - View button of description column in Statement table not supported by websocket API yet
 - Newer and Older button of Statement table not completed yet

### CODE REVIEW

**Comment by ashkanx** [comment]
Can you deploy it to your QA please?

**Comment by ashkanx** [comment]
Can you deploy it to your QA please?

File: Gruntfile.js [hunk 1]
```diff
@@ -10,4 +10,21 @@ module.exports = function (grunt) {
             scope: 'devDependencies'
         }
     });
+    //
+    //grunt.initConfig({
```
Comment (raunakkathuria) [inline Gruntfile.js]: if its not needed please remove it rather than commenting

File: _SpecRunner.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!doctype html>
```
Comment (raunakkathuria) [inline _SpecRunner.html]: why this file is needed?

File: config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "website_name": "Binary.com",
     "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR,ZH_TW,VI",
     "static": {
-        "url": "https://static.binary.com/",
+        "url": "https://qingweibinary.github.io/binary-static/",
```
Comment (raunakkathuria) [inline config.json]: this should not go live

Comment (qingweibinary) [inline config.json]: Yes, a serious mistake, sorry

**Comment by qingweibinary** [comment]
I will amend those mistake and resubmit

**Pull request closed without merging**